docs/developer-docs/security/rust-canister-development-security-best-practices.md

```diff
@@ -425,6 +425,29 @@ If a canister traps or panics in `pre_upgrade`, this can lead to permanently blo
 
 - See [Current limitations of the Internet Computer](https://wiki.internetcomputer.org/wiki/Current_limitations_of_the_Internet_Computer), section "Bugs in `pre_upgrade` hooks"
 
+### Reinstantiate global timers during upgrades
```

Comment:
Would it make sense to drop the word "global" because it sounds confusing?

On the system API level, we have a single global timer (that's why it is called global). Motoko and Rust CDK implement support for many user timers on top of the global timer.

Reply:
```suggestion
### Reinstantiate timers during upgrades
```